src/components/CustomSideBar.tsx

```diff
@@ -27,16 +27,18 @@ const CustomSideBar = (props: CustomSideBarProps) => {
         blockScroll
         className={`custom-sidebar relative ${
           props.themeClasses ? props.themeClasses : ""
-        }`}
+        }  ${props.sideBarWidthClass ? props.sideBarWidthClass : "!w-50w"}`}
```

Comment:
How about  `${props.sideBarWidthClass || "!w-50w"}` ?
That's less code.

Reply:
done